enterprise/app/BUILD.bazel

```diff
@@ -105,6 +108,14 @@ ts_library(
     ],
 )
 
+js_library(
```

Comment:
not a big deal since it's such a small amount of code but does this have to be JS or could we use TS for this too?

Reply:
heh, wasn't planning on mailing this just yet, but I did this because of typescript whining about the import from monaco (maybe there's a types package i missed?)